DSharpPlus/Entities/Emoji/DiscordEmoji.EmojiUtils.cs

```diff
@@ -10144,7 +10145,8 @@ static DiscordEmoji()
                 ["\U0001f9df"] = ":zombie:",
                 ["\U0001f4a4"] = ":zzz:",
             };
-            #endregion
+
+            #endregion Generated Emoji Map
```

Comment:
Not necessary

Reply:
That's my IDE autocomplete kicking in, I can remove it